store/retrylayer/retrylayer.go

```diff
@@ -4776,21 +4776,63 @@ func (s *RetryLayerTeamStore) AnalyticsGetTeamCountForScheme(schemeId string) (i
 
 }
 
-func (s *RetryLayerTeamStore) AnalyticsPrivateTeamCount() (int64, *model.AppError) {
+func (s *RetryLayerTeamStore) AnalyticsPrivateTeamCount() (int64, error) {
 
-	return s.TeamStore.AnalyticsPrivateTeamCount()
+	tries := 0
```

Comment:
@jespino - Why does changing the error type generate this code? Shouldn't it be already there?

Reply:
We are only able to retry if this is a plain error, we can't retry an AppError because the AppError doesn't provide the underneath pq or mysql error to verify if it is a repeatable error or not. So, if you don't return a plain error, I just not even try to retry.

Reply:
Ah yes, of course. Thanks.